src/test/FundingOracles.t.sol

```diff
@@ -142,10 +146,8 @@ contract FundingOraclesTest is BaseFixture {
 
         vm.startPrank(keeper);
         medianOracleWbtc.pushReport(1000);
-        // TODO: For some reason, the revert string is not being thrown and the trace is wrong. 
-        //       Maybe a bug in forge?
-        vm.expectRevert();
-        skip(1 days + 1);
+        vm.warp(1 days + 1);
+        vm.expectRevert("price must not be zero"); // When expired returns 0, valid = false
```

Comment:
I'd keep this as `skip` since `vm.warp` sets the `block.timestamp` instead of skipping ahead with a duration
https://github.com/foundry-rs/forge-std/blob/ea7c38a50f5f74f35893cc68aa176bcc565a923a/src/Vm.sol#L6-L7
https://github.com/foundry-rs/forge-std/blob/master/src/Test.sol#L22-L25

Reply:
Makes total sense, will switch it back

Reply:
You are right that the revert messages are not returned here, the reason why I was seeing it with vm.warp is that I was going to a weird off timestamp that was producing a 0 price, hence the revert.